RELEASE/scripts/autoscend/auto_mr2018.ash

```diff
@@ -2,14 +2,13 @@ script "auto_mr2018.ash"
 
 #	This is meant for items that have a date of 2018.
 
+boolean isjanuaryToteAvailable(){
+	return item_amount($item[January\'s Garbage Tote]) > 0 && is_unrestricted($item[January\'s Garbage Tote]);
```

Comment:
I thought we were supposed to use auto_is_valid()?

Reply:
yep, good catch